src/enzo/FindSubgrids.C

```diff
@@ -67,18 +67,18 @@ int FindSubgrids(HierarchyEntry *Grid, int level, int &TotalFlaggedCells,
   if (CurrentGrid->SetFlaggingField(NumberOfFlaggedCells, level) == FAIL) {
     ENZO_FAIL("Error in grid->SetFlaggingField.");
   }
- 
+
+  /* Flag cells based on MultiRefineRegions */
+
+  if (CurrentGrid->SetFlaggingFieldMultiRefineRegions(level)
+      == FAIL) {
+    fprintf(stderr, "Error in grid->SetFlaggingFieldMultiRefineRegions.\n");
+    return FAIL;
+  }
+
```

Comment:
This is probably OK, but why did this get moved outside of the `NumberOfFlaggedCells !=0` if statement?

Reply:
This is done because there's a conflict between MustRefineParticles and MultiRefineRegions.  The MustRefineParticles are set up to derefine everywhere outside of their area, so if you want to force refinement in that area you have to do it separately.  That's why this is here.

Reply:
reading through, it doesn't look like it should've been in there in the first place?